lib/promutils/time.go

```diff
@@ -2,11 +2,18 @@ package promutils
 
 import (
 	"fmt"
+	"math"
 	"strconv"
 	"strings"
 	"time"
 )
 
+var (
+	// Time.UnixNano can only store maxInt64, which is 2262
```

Comment:
Any reason, why we should use variables instead of

```go
const(
  maxValidYear = 2262
  minValidYear = 1970
)
```

 Also, I suggest to use `strconv.Atoi` with care. It may work incorrectly with big numbers at x86 arch. strconv.ParseInt is safe approach.